tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
:+1: IIRC, I think even [NLTK](http://www.nltk.org/) does the same kind of thing, the dictionaries are separate from the actual toolkit itself and have to be downloaded separately before NLTK can use them.